net/openthread-br/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+#  Copyright (c) 2021, The OpenThread Authors.
+#  All rights reserved.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openthread-br
+PKG_RELEASE:=1
+PKG_VERSION:=0.3.0
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/openthread/ot-br-posix.git
+PKG_SOURCE_DATE:=2021-04-23
+PKG_SOURCE_VERSION:=5a3c3221d6ae6b087d0933d98657ba818ae6ff31
+PKG_MIRROR_HASH:=skip
```

Comment:
Why?

Reply:
Do you mean why set `skip`? I explained in https://github.com/openwrt/packages/pull/15496#issuecomment-826483763.

I found checking failures in github actions when set it through:

```bash
make package/openthread-br/check FIXUP=1 V=s
```